wordpress/wp-admin/includes/menu.php

```diff
@@ -164,17 +164,23 @@
 		$_wp_menu_nopriv[ $data[2] ] = true;
 	}
 
-	/*
-	 * If there is only one submenu and it is has same destination as the parent,
-	 * remove the submenu.
-	 */
-	if ( ! empty( $submenu[ $data[2] ] ) && 1 === count( $submenu[ $data[2] ] ) ) {
-		$subs      = $submenu[ $data[2] ];
-		$first_sub = reset( $subs );
-		if ( $data[2] == $first_sub[2] ) {
-			unset( $submenu[ $data[2] ] );
-		}
-	}
+    // ===BEIS SEA===
+    // The code below is deliberately commented out by us as a work around to a WordPress bug that was preventing
+    // the proper configuration of the Local Authority role.
+    // The bug stops Local Authority users from seeing the list of Local Authorities (error displayed).
+    // See:https://stackoverflow.com/questions/58218457/wordpress-user-with-custom-role-cannot-view-list-page-for-custom-post-types-with/58234091
+
+    /*
+     * If there is only one submenu and it is has same destination as the parent,
+     * remove the submenu.
+     */
+    //	if ( ! empty( $submenu[ $data[2] ] ) && 1 === count( $submenu[ $data[2] ] ) ) {
+    //		$subs      = $submenu[ $data[2] ];
+    //		$first_sub = reset( $subs );
+    //		if ( $data[2] == $first_sub[2] ) {
+    //			unset( $submenu[ $data[2] ] );
+    //		}
+    //	}
```

Comment:
This fix was overwritten by our WordPress installation, so we had to add it again.
This solves an issue where on the admin site, if you were a local authority user, you couldn't open the local authority posts.

Reply:
According to Stackoverflow, there's a fix we could go with here which doesn't involve changing the core files: https://stackoverflow.com/a/60391723/2573402 (the bottom answer)

I'd be in favour of switching to that if it's simple (just because I think this is the only change to the core files?)

Reply:
Sounds good to me, I'll take a look

Reply:
After investigating further, neither of the suggested alternative fixes offer a perfect solution. Given that the current fix has been fully tested and has not caused any issues, we're going to keep it in (even though it's not idea that it updates a wp core file). Hopefully a future release may remove the need for this, but not yet.

This is documented already here: documentation\Operational Support.md
![image](https://user-images.githubusercontent.com/71124134/115381876-ed80b400-a1cb-11eb-81c6-362ae2b2ff37.png)